cwl/src/main/scala/cwl/TypeAliases.scala

```diff
@@ -97,6 +97,12 @@ trait TypeAliases {
   type ScatterVariables = Option[String :+: Array[String] :+: CNil]
 }
 
+object CwlAny {
+  implicit class EnhancedCwlAny(val cwlAny: CwlAny) extends AnyVal {
+    def stringRepresentation: String = cwlAny.select[Json] flatMap { json => json.asString } getOrElse "CwlAny (not Json)"
```

Comment:
sorry if this is a dumb q but can this really not be Json?

Reply:
I'm following the `CwlAny` type def. I suspect not (hence why I don't really bother trying to make it a string)